content/es/style-guide/_index.md

```diff
@@ -159,8 +155,6 @@ Cuando corresponda, use **ejemplos del mundo real** que ayuden a los lectores (e
 
 Cuando se use en su definición, siempre **enlace a los términos existentes del glosario** (solo la primera mención debe tener un hipervínculo).
 
-**Ejemplo**: eche un vistazo a la sección "Qué es" de la [definición de service mesh](/es/service-mesh/).
-Se vincula con las definiciones de microservicios, servicio, fiabilidad y observabilidad.
```

Comment:
I understand why you preferred to remove these lines, but, after checking on the EN source ([ref](https://github.com/cncf/glossary/blob/main/content/en/style-guide/_index.md?plain=1#L161-L162)) I see that what they did is to rephrase the sentence using "summary" to refer to the section instead. So, what do you think about applying this change instead?

```markdonw
**Ejemplo**: eche un vistazo a la introducción de la [definición de service mesh](/es/service-mesh/).
Se vincula con las definiciones de microservicios, servicio, fiabilidad y observabilidad.
```

Worth noting that "introducción" is not a direct translation for "summary", but I think it applies better in that context. Still, I think it is open to discussion.

Reply:
I agree with you.